src/open_inwoner/plans/tests/test_views.py

```diff
@@ -561,6 +564,113 @@ def test_plan_action_status_not_your_action(self):
         )
         self.assertEqual(response.status_code, 404)
 
+    @freeze_time("2022-01-01")
+    def test_plan_list_renders_expected_data_when_begeleider(self):
+        self.user.contact_type = ContactTypeChoices.begeleider
+        self.user.save()
+        self.plan.end_date = date(2022, 1, 20)
+        self.plan.save()
+
+        response = self.app.get(self.list_url, user=self.user)
+        rendered_plan_title = response.pyquery("tbody .table__header")[0].text
+        rendered_contact = response.pyquery("tbody .table__item")[0].text
+        rendered_end_date = response.pyquery("tbody .table__item")[1].text
+        rendered_plan_status = response.pyquery("tbody .table__item")[2].text
+        rendered_actions_num = response.pyquery("tbody .table__item")[3].text
+        rendered_action_required = response.pyquery(
+            "tbody .table__item--notification-danger"
+        )[0].text
+
+        self.assertEqual(rendered_plan_title, self.plan.title)
+        self.assertEqual(rendered_contact, self.contact.get_full_name())
+        self.assertEqual(rendered_end_date, "20-01-2022")
+        self.assertEqual(rendered_plan_status, _("Open"))
+        self.assertEqual(rendered_actions_num, "1")
+        self.assertEqual(rendered_action_required, _("Actie vereist"))
+
+    @freeze_time("2022-01-01")
+    def test_plan_list_renders_expected_data_for_expired_plan_when_begeleider(self):
+        self.user.contact_type = ContactTypeChoices.begeleider
+        self.user.save()
+        self.plan.end_date = date(2022, 1, 1)
+        self.plan.save()
+
+        response = self.app.get(self.list_url, user=self.user)
+        rendered_end_date = response.pyquery("tbody .table__item")[1].text
```

Comment:
I see a lot of indexes into the pyquery result list (plain integer indexes are almost magic numbers so not great).

It is fine for now but not so great to edit or debug later.